dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -1312,24 +1313,71 @@ sub get_aliases {
 
   # Parse alias definitions
   # format is
-  #    alias = value alias = value ...
-
-  while ($line =~ s/^\s*([\w:]+)\s*=\s*(\w+)\s*//) {
-    my ($alias, $value) = ($1, $2);
+  #    alias = value Pack::alias = value ...
+  # or
+  #    alias => other
+  # or
+  #    alias => Pack::other
+  # or
+  #    Pack::alias => Other::alias
+
+  while ($line =~ s/^\s*([\w:]+)\s*=(>?)\s*([\w:]+)\s*//) {
+    my ($alias, $is_symbolic, $value) = ($1, $2, $3);
     my $orig_alias = $alias;
 
+    blurt( $self, "Error: In alias definition for '$alias' the value may not"
+                  . " contain ':' unless it is symbolic.")
+        if !$is_symbolic and $value=~/:/;
+
```

Comment:
ParseXS is also used for C++ where a constant name containing `::` is normal.

Reply:
Really not sure what you want me to do here. Add the enhancement to the PR since we are doing a version bump anyway? Or create a new PR? As you said, currently we dont support this form of value name at all, it would have to be wrapped in a define.

Reply:
You can ignore this.

I could see ALIAS supporting C++ symbols, or even general constant expressions, eg:
```
ALIAS:
  get_alpha: TAG_ALPHA * 2
  set_alpha: TAG_ALPHA * 2 + 1
...
```
but there's no particular reason to include it in this PR.

Reply:
FWIW, this PR includes doc changes that show how to work around any limitations in this area. As long as these expressions are hidden behind a define they are fine.